pkg/proxy/iptables/proxier.go

```diff
@@ -1605,6 +1605,8 @@ func (proxier *Proxier) syncProxyRules() {
 	proxier.iptablesData.Write(proxier.natRules.Bytes())
 
 	klog.V(5).InfoS("Restoring iptables", "rules", proxier.iptablesData.Bytes())
+	numberIptablesRules := utilproxy.CountBytesLines(proxier.iptablesData.Bytes())
```

Comment:
This will include COMMIT lines and chain definitions.  Is that what we want in this metric?  It may not matter much, but I want to be clear

Reply:
Maybe we should count lines in proxier.filterRules.Bytes() and proxier.natRules.Bytes() instead?

Reply:
My use case was that I wanted to see the graph NetworkProgrammingLatency/IptablesRulesTotal, so I took the short path but :thinking: , now that you mention this, what happen if someone is using some cni plugin/nework policy thing that install rules? it will affect kube-proxy and you'll see only that the rules grow.

Let me change it and tag the metric with filter/nat:

Reply:
I need coffee, this only considers kube-proxy rules not all the hosts rules, but this may be used later to optimize kube-proxy and the change is simple

Reply:
changed, I still have to test it

Reply:
test done, I didn't remember that you can unit test the metrics.
The test could be much better, but I think that this is good enough